docs/anonymous-statistics.md

```diff
@@ -103,7 +103,10 @@ Each of these opt-out processes does the following:
 
 -   Prevents the daemon from executing the anonymous statistics script.
 -   Forces the anonymous statistics script to exit immediately.
--   Stops the Google Tag Manager Javascript snippet, which remains on the dashboard, from firing and sending any data to
-    Google Analytics.
+-   Stops the PostHog Javascript snippet, which remains on the dashboard, from firing and sending any data to the Netdata PostHog.
+
+## Migration from Google Analytics and Google Tag Manager.
+
+Prior to vX.XX we used Google Analytics to capture this information. This led to discomfot with some of our users in sending any product usage data to a third party like Google. It was also not even that useful in terms of generating the insights we needed to help catch bugs early and find opportunities for product improvement as Google Analytics does not allow its users access to the raw underlying data without paying a significant amount of money which would be infeasible for a project like Netdata.
 
-[![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Fdocs%2Fanonymous-statistics&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)]()
+While we migrate fully away from Google Analytics to PostHog there maybe be a small period of time where we run both in parallel before we remove all Google Analytics related code. This is to ensure we can fully test and validate the Netdata PostHog implementation before fully defaulting to it.
```

Comment:
Even though it's sort of ironic, I believe the GA tag at the end (the `[!analytics](...` stuff) should remain, since we're using it to track pageviews, and it's sort of orthogonal to the (awesome) work you're doing in migrating the Agent anonymous statistics.

Reply:
yep - have been seeing that in the collector docs too. I think we probably can/cloud move this stuff into its own PostHOg project at a later stage if we want to. But agree should stay for now.